examples/05-plotting/01-compare_results.py

```diff
@@ -0,0 +1,48 @@
+"""
+.. _compare_results:
+
+Compare results using the plotter
```

Comment:
Consider using capitalized titles.

```suggestion
Compare Results Using the Plotter
```